pygmt/src/grdclip.py

```diff
@@ -138,6 +138,11 @@ def grdclip(
     Full option list at :gmt-docs:`grdclip.html`
 
     {aliases}
+       - G = outgrid
+       - Sa = above
+       - Sb = below
+       - Si = between
+       - Sr = replace
```

Comment:
The `{aliases}` placeholder will be expanded to something like:
```
.. hlist:: 
   :columns: 3
   
   - R = region
   - J = projection
```
Currently, the solution is to add the missing aliases below the `{aliases}` placeholder. Does it looks bad? Perhaps we should maintain the full list of aliases manually, i.e., remove the `{aliases}` placeholder and manually add all aliases.

Reply:
> Perhaps we should maintain the full list of aliases manually, i.e., remove the `{aliases}` placeholder and manually add all aliases.

See the changes in `grdfill.py` for another style.